### ISSUE TITLE
usb_cam is missing in ros install?

### ISSUE THREAD

**Issue by yaromins** [OpenAgInitiative/openag_brain_install_rpi#3]
I was trying to add usb camera to my setup and got an error. 
Seems that usb_cam module is not installed during initial setup:
 [`rosinstall_generator ros_comm rosserial rosserial_arduino --rosdistro indigo --deps --wet-only --exclude roslisp --tar > indigo.rosinstall`](https://github.com/OpenAgInitiative/openag_brain_install_rpi/blob/master/install.sh#L28)

**Comment by yaromins** [comment]
Ok, I have found that usb_cam is installed in separate repository with a bit confusing name [openag_ros](https://github.com/OpenAgInitiative/openag_ros)

**Comment by yaromins** [comment]
According to this [explanation](http://forum.openag.media.mit.edu/t/installing-openag-ros/447/2?u=serein) openag_ros was transitional package and now is outdated, so I think usb_cam installation is indeed missing. @LeonChambers please comment

**Comment by LeonChambers** [comment]
Yes, this script probably should install `usb_cam`. I just hadn't gotten around to adding it yet.
